src/main/java/org/opentripplanner/apis/gtfs/datafetchers/VehicleRentalStationImpl.java

```diff
@@ -96,6 +98,40 @@ public DataFetcher<Integer> vehiclesAvailable() {
     return environment -> getSource(environment).getVehiclesAvailable();
   }
 
+  @Override
+  public DataFetcher<Iterable<VehicleTypeCount>> vehicleTypesAvailable() {
+    return environment -> {
+      VehicleRentalPlace rentalPlace = getSource(environment);
+      if (rentalPlace instanceof VehicleRentalStation) {
```

Comment:
If you change the type below (it's currently wrong) then you don't need the `instanceOf`.